specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-03-31-preview",
+    "title": "agfood",
+    "description": "Description of the new service",
+    "x-ms-code-generation-settings": {
+      "name": "agfoodClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [
+    {
+      "azure_auth": ["user_impersonation"]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AgFoodPlatform/operations": {
+      "get": {
+        "tags": ["Tag1"],
+        "operationId": "OperationGroup_Get",
+        "x-ms-examples": {
+          "BatchAccountDelete": { "$ref": "./examples/OperationGroupGet.json" }
+        },
+        "description": "This is a sample get operation, please see guidelines in azure-rest-api-specs repository for more info",
```

Comment:
Can you please clarify the intent of this PR?  This looks like some kind of sample content.

Reply:
Joel,
Sending details in a mail. We are onboarding an new service onto Azure. Have updated the PR. Request your help to review the same.